Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM node:latest
 
-ADD . /app
+RUN mkdir -p /app
 WORKDIR /app
 
+COPY package.json /app/
 RUN npm install
+
+COPY . /app
+
+CMD [ "./zenbot.sh", "trade" ]
```

Comment:
Yes, the default command for the bot to run is running `./zenbot trade`. However it can be overwritten in `docker-compose.yml` or when running as a docker run command.